### DISCUSSION TITLE
Error: Unexpected 'wght' value while compiling variable font

### DISCUSSION THREAD

**Discussion by ahmadfawaid** [rsms/inter#764] [General]
**Describe the bug**
Changed the weight and updated the Axis Mappings in Font Info, but got an unexpected weight error when creating the variable font

**To Reproduce**
1. Open the glyphs package file and change the Semibold weight from 580 to 590 (or another value), update the Axis Mappings as well.
2. Run `make -j var`

**Screenshots**
<img width="604" alt="Screenshot 2024-06-17 at 7 36 17 AM" src="https://github.com/rsms/inter/assets/16203145/3005f741-b652-46f2-a8bd-ca383694249d">
<img width="310" alt="Screenshot 2024-06-17 at 7 36 38 AM" src="https://github.com/rsms/inter/assets/16203145/bf186936-0f7b-4c35-a944-b35ad695cfc1">
<img width="626" alt="Screenshot 2024-06-17 at 7 28 42 AM" src="https://github.com/rsms/inter/assets/16203145/6f80875c-928d-4a3b-9725-d86b1fea52a5">

**Environment**
- OS: macOS 14.1
- Version of font: Inter 4.0

**Comment by rsms** [comment]
Why did you "Changed the weight and updated the Axis Mappings in Font Info"?
(If you're making changes to the source files beyond fixing issues, you're on your own I'm afraid.)

**Comment by ahmadfawaid** [comment]
I usually use variables to adjust the font weight for my personal design projects. However, this time, I plan to manually update the font weight, export the new static font, and send it to the dev team. Sometimes, the Bold weight feels too heavy, so I reduce it to 670 or 680. Similarly, the SemiBold weight can also feel too bold, so I adjust it to 580.

Note: This is only for my personal project, and I don't intend to make any changes to the source files of the Inter font. If the issue I’ve sent is not relevant, feel free to disregard it.

**Comment by rsms** [comment]
I see. Let's move this to discussions as it's not a bug
